Gemfile

```diff
@@ -22,4 +22,18 @@ else
   gem 'puppet', :require => false
 end
 
+group :system_tests do
+  if beaker_version = ENV['BEAKER_VERSION']
+    gem 'beaker', *location_for(beaker_version)
+  end
+  if beaker_rspec_version = ENV['BEAKER_RSPEC_VERSION']
+    gem 'beaker-rspec', *location_for(beaker_rspec_version)
+  else
+    gem 'beaker-rspec',  :require => false
+  end
+  gem 'serverspec',    :require => false
+  gem 'beaker-puppet_install_helper', :require => false
+  gem 'master_manipulator', '1.1.2', :require => false
+end
+
 # vim:ft=ruby
```

Comment:
I think this module is uses module-sync so you should make changes directly to the Gem file.